lib/CFRMessageProvider.jsm

```diff
@@ -345,7 +343,6 @@ const CFR_MESSAGES = [
     frequency: {lifetime: 3},
     targeting: `
       localeLanguageCode == "en" &&
-      (providerCohorts.cfr == "three_per_day") &&
```

Comment:
I'm a bit confused with this cohort business.. how come some of them still have cohorts but the config pref doesn't have cohort anymore?

Reply:
I was just going to leave the cohort in case we wanted to test more with it, I guess I could just remove it though